tests/test_interface.rb

```diff
@@ -471,7 +476,7 @@ def test_interface_description_valid
     description = 'This is a test description ! '
     interface.description = description
     assert_equal(description.rstrip, interface.description)
-    interface_ethernet_default(interfaces_id[0])
+    interface_ethernet_default(interfaces[0])
```

Comment:
Redundant now that you're doing this in teardown(), isn't this?

Same comment below.